docs/api/scala/index.md

```diff
@@ -1,11 +1,20 @@
 # MXNet - Scala API
+MXNet supports the Scala programming language. The MXNet Scala package brings flexible and efficient GPU
+computing and state-of-art deep learning to Scala. It enables you to write seamless tensor/matrix computation with multiple GPUs in Scala. It also lets you construct and customize the state-of-art deep learning models in Scala, and apply them to tasks, such as image classification and data science challenges.
 
-See the [MXNet Scala API Documentation](http://mxnet.incubator.apache.org/api/scala/docs/index.html).
+See the [MXNet Scala API Documentation](https://mxnet.incubator.apache.org/api/scala/docs/index.html#ml.dmlc.mxnet.package) for detailed API information.
 
-MXNet supports the Scala programming language. The MXNet Scala package brings flexible and efficient GPU
-computing and state-of-art deep learning to Scala. It enables you to write seamless tensor/matrix computation with multiple GPUs in Scala. It also lets you construct and customize the state-of-art deep learning models in Scala,
-  and apply them to tasks, such as image classification and data science challenges.
 
+## Image Classification with the Scala Infer API
+The Infer API can be used for single and batch image classification. More information can be found at the following locations:
+
+* [Infer API Overview](https://mxnet.incubator.apache.org/api/scala/infer.html)
+* [Infer API Scaladocs](http://mxnet.incubator.apache.org/versions/master/api/scala/docs/index.html#ml.dmlc.mxnet.infer.package)
```

Comment:
Link should point to org.apache.mxnet.infer.package now.
Namespace change PR from ml.dmlc.mxnet -> org.apache.mxnet is merged.